ksp_plugin_adapter/burn_editor.cs

```diff
@@ -56,29 +56,45 @@ class BurnEditor : ScalingRenderer {
     ComputeEngineCharacteristics();
   }
 
+  public enum Event {
+    None,
+    Changed,
+    Deleted,
+    Minimized,
+    Maximized,
+  }
+
   // Renders the |BurnEditor|.  Returns true if and only if the settings were
   // changed.
-  public bool Render(string header,
+  public Event Render(string header,
                      bool anomalous,
-                     double burn_final_time,
-                     Action delete) {
+                     double burn_final_time) {
     bool changed = false;
```

Comment:
The code would be a bit cleaner if you removed this `bool` and had a local variable of type `Event`.

Reply:
We need a `bool` for the `|=`. We should probably refactor this quite a bit, in particular so that it doesn’t return `changed` on the first repaint, but not today.